InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -398,20 +409,32 @@ class InMobiAdapter : PartnerAdapter {
                         return Result.failure(ChartboostMediationAdException(ChartboostMediationError.CM_LOAD_FAILURE_INVALID_BANNER_SIZE))
                     }
 
-                    return suspendCoroutine { continuation ->
+                    return suspendCancellableCoroutine { continuation ->
                         // Load an InMobiBanner
-                        InMobiBanner(activity, placement).apply {
+                        inMobiBannerAds[request.identifier] = InMobiBanner(activity, placement)
+                        inMobiBannerAds[request.identifier]?.apply {
```

Comment:
why not this `apply` this to the previous line? That way it's guaranteed to not be null

Reply:
Wanted to share something that was working before doing the other adjustments. Hope nothing new pops up. 😨